framework/sysdeps/unix/child_debug.cpp

```diff
@@ -133,11 +133,11 @@ struct CrashContext
 }
 
 enum CrashAction : uint8_t {
+    kill_on_crash           = 0x00,
```

Comment:
Is there a good reason to define it as 0? Now it will become the only field that cannot be tested with
`if (crash_action & kill_on_crash) { ... }` and has to be tested with `==`.
I am worried that we might shoot ourselves in the foot at some point.

Reply:
Because "kill" is the absence of any other action. We don't let it dump core, we don't ask for a context, we don't ask for a backtrace and we won't attach gdb.

Reply:
Yeah... `kill_on_crash` was already handled uniquely, so blocking this particular PR doesn't really make it better.